src/generator/default/transformer.php

```diff
@@ -0,0 +1,50 @@
+<?php
+/**
+ * @var string $namespace
+ * @var \cebe\yii2openapi\lib\items\Transformer $transformer
+*/
+use yii\helpers\Inflector;
+
+echo '<?php';
+?>
+
+namespace <?= $namespace ?>;
+
+use League\Fractal\TransformerAbstract;
+use <?=$transformer->modelFQN?>;
+
+class <?=$transformer->name?> extends TransformerAbstract
+{
+<?php if (!empty($transformer->availableRelations)):?>
+     protected $availableIncludes = ['<?=implode("', '", $transformer->availableRelations)?>'];
+<?php else:?>
+     protected $availableIncludes = [];
+<?php endif;?>
+<?php if (!empty($transformer->defaultRelations)):?>
+     protected $defaultIncludes = ['<?=implode("', '", $transformer->defaultRelations)?>'];
+<?php else:?>
+     protected $defaultIncludes = [];
+<?php endif;?>
+
+     public function transform(<?=$transformer->dbModel->getClassName()?> $model)
+     {
+          return $model->getAttributes([]);
```

Comment:
@Insolita Is it correct to pass empty array here? `null` means all attributes, `[]` results in no attributes to be serialized.

Reply:
Technically is correct, and should return an empty array. But really it is not that I wanted. Fixed